metrumresearchgroup/lastdose
# Features and fixes

# Summary
- add option `lastdose.include_tafd`
- add option `lastdose.id_col`
- add option `lastdose.time_col`
- handle missing values in time column 
- `include_tafd` now defaults to `FALSE`

- tests for handling missing values in TIME
- search for ID column using `find_id_col()`
- search for time column using `find_time_col()`

See  #25, #27, #29, #30
